packages/style-engine/src/styles/utils.ts

```diff
@@ -127,7 +127,12 @@ export function generateBoxRules(
 }
 
 /**
- * Returns a CSS var value from incoming style value following the pattern `var:description|context|slug`.
+ * Returns a WordPress CSS custom var value from incoming style preset value.
```

Comment:
Side note: I think the style engine in general could subsume a lot of style utility functions, even in the backend. E.g., ref resolution, CSS var building and more. 

I see the benefit as reducing `WP_Theme_JSON` bloat and grouping 'like' concerns.